sdk/samples/iot/hub/src/paho_iot_hub_sas_telemetry_example.c

```diff
@@ -175,7 +195,7 @@ static az_result read_configuration_and_init_client()
 
   iot_hub_sas_key_span = AZ_SPAN_FROM_BUFFER(iot_hub_sas_key);
   AZ_RETURN_IF_FAILED(read_configuration_entry(
-      "IoT Hub Device SAS Key",
+      ENV_IOT_HUB_SAS_KEY,
```

Comment:
:+1:

Reply:
haha already had this change made for my next push :D yas! 💯

Reply:
nm last comment. i got myself confused.